docs/core/settings/variables.md

```diff
@@ -27,24 +30,102 @@ You can use special variables in several places:
 * [[link,auth_ldap]], [[link,auth_sql]], and [[link,userdb]] query strings
 * Log prefix for imap/pop3 process
 
-## Global Variables
-
-Global variables that work everywhere are:
+## Variable expansion syntax
+
+In Dovecot 2.4/3.1 we have introduced a totally new variable expansion syntax.
```

Comment:
Shouldn't this be using `[[changed,var_expand_syntax]]` or something, rather than specifying explicit version numbers?

Reply:
Also I think s/a totally/an entirely/ is a bit better.